### ISSUE TITLE
The Cloud Firestore emulator is not running

### ISSUE THREAD

**Issue by rublev** [firebase/firebase-tools#2809]
### [REQUIRED] Environment info

firebase-tools: v8.15.1

Platform: macOS

### [REQUIRED] Test cases

Start emulators. Start function shell.

### [REQUIRED] Steps to reproduce

Really unsure. I ran `firebase emulators:start --only functions,firestore` and it works fine, the emulators I mean. Then I run `firebase functions:shell` and test my functions and functions do not update when I save them.

### [REQUIRED] Expected behavior

For the functions shell to update my functions when I update them after running `firebase:shell` and for it to detect that my emulator is running locally.

### [REQUIRED] Actual behavior

Output of `firebase functions:shell`:
```

$ firebase functions:shell
⚠  functions: Your GOOGLE_APPLICATION_CREDENTIALS environment variable points to /Users/user/dev/project/app/serviceKey.json. Non-emulated services will access production using these credentials. Be careful!
⚠  functions: The Cloud Firestore emulator is not running, so calls to Firestore will affect production.
i  functions: Loaded functions: ..., nuxtssr
i  functions: Connected to running firestore emulator at localhost:8080, calls to this service will affect the emulator
⚠  functions: The following emulators are not running, calls to these services will affect production: database, pubsub
```

**Comment by rublev** [comment]
> This issue does not seem to follow the issue template. Make sure you provide all the required information.

How not? I basically just copied https://github.com/firebase/firebase-tools/issues/2367 which is identical to my issue.

**Comment by rublev** [comment]
Identical issue: https://github.com/firebase/firebase-tools/issues/2116

**Comment by samtstern** [comment]
@rublev the bot was confused because you renamed the "Test case" header to "Test cases".   It's not too smart :-)

**Comment by rublev** [comment]
My bad.

**Comment by rublev** [comment]
functions code: 

```javascript
const admin = require('firebase-admin')

const functions = require('firebase-functions')

exports.testFunction2 = functions.https.onCall((data, context) => {
  console.log('test')
  return { test1: true }
})
```
No matter what code I changed in my function, I get:

```
firebase > >  {"severity":"WARNING","message":"Request body is missing data."}
>  {"severity":"ERROR","message":"Invalid request, unable to process."}

RESPONSE RECEIVED FROM FUNCTION: 400, {
  "error": {
    "message": "Bad Request",
    "status": "INVALID_ARGUMENT"
  }
}
```

If I change the function name to testFunction3 or anything else, it updates in the emulator a la: `✔  functions[test-testFunction2]: http function initialized (http://localhost:5001/proj-1234/us-central1/test-testFunction2).` but it always yields the same error and never prints the log. Same when I visit the `5001/proj-1234/us-central1/test-testFunction2` URL, same output every time:

```javascript
{
error: {
message: "Bad Request",
status: "INVALID_ARGUMENT"
}
}
```

**Comment by rublev** [comment]
Turns out everything is fine. All I had to do was call it with `test.testFunction3('')` instead of `test.testFunction3()` like before. Why do I have to add data to the parameter now magically to get it working? I have no idea, nowhere did I see this in a changelog or anything. Yesterday it stopped working, now adding an empty string param has it executing just fine.

Love losing 2 days for zero reason. Lovely.
